CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# How to contribute
+
+WIP
+
+## Clone the repo
+
+```
+> git clone https://github.com/cockroachdb/ui/
+```
+
+The UI repo is a monorepo managed by [Lerna](https://lerna.js.org/). Once you clone the repo, be sure to [bootstrap](). _In the following examples, I'm using `npx`. If you do this very often you may consider installing Lerna globally (`npm i -g lerna`)._
```

Comment:
nit: looks like there is a link missing at `[bootstrap]()`

Reply:
good catch.